qa/L0_multi_gpu/test.sh

```diff
@@ -0,0 +1,137 @@
+# Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+source ../common/util.sh
+
+rm -f *.log
+
+# Set test parameters
+MODEL_ANALYZER="`which model-analyzer`"
+MODEL_REPOSITORY=${MODEL_REPOSITORY:="/mnt/dldata/inferenceserver/model_analyzer_clara_pipelines"}
+MODEL_NAMES="classification_chestxray_v1"
+BATCH_SIZES="4"
+CONCURRENCY="4"
+PORTS=(`find_available_ports 3`)
+http_port="${PORTS[0]}"
+grpc_port="${PORTS[1]}"
+metrics_port="${PORTS[2]}"
+MODEL_ANALYZER_BASE_ARGS="-m $MODEL_REPOSITORY -n $MODEL_NAMES -b $BATCH_SIZES -c $CONCURRENCY"
+MODEL_ANALYZER_PORTS="--triton-http-endpoint localhost:$http_port --triton-grpc-endpoint localhost:$grpc_port"
+MODEL_ANALYZER_PORTS="$MODEL_ANALYZER_PROTS --triton-metrics-url http://localhost:$metrics_port/metrics"
+TRITON_LAUNCH_MODES="docker remote local"
+TRITON_SERVER_VERSION="20.11-py3"
+CLIENT_PROTOCOLS="http grpc"
+TEST_OUTPUT_NUM_ROWS=1
+TEST_OUTPUT_NUM_COLUMNS=7
+
+# Run the model-analyzer, both client protocols
+RET=0
+
+function check_gpus() {
+    ananlyzer_gpus=($@)
+    gpu_uuids=(`get_all_gpus_uuids`)
+    cuda_devices=(`echo $CUDA_VISIBLE_DEVICES | sed "s/,/ /g"`)
+
+    if [ ${#ananlyzer_gpus[@]} != ${#cuda_devices[@]} ]; then
+        return 1
+    fi
+
+    index=0
+    for cuda_index in $cuda_devices; do
+        gpu_uuid=${gpu_uuids[$cuda_index]}
+        if [ $gpu_uuid != ${ananlyzer_gpus[$index]} ]; then
+            echo -e "\n***\n*** Model Analyzer is not using the correct GPUs.\n***"
+            return 1
+        fi
+        index=$((index+1))
+    done
+    return 0
+}
+
+function run_server_luanch_modes() {
```

Comment:
Is this a refactor of L0_server_launch_modes/test.sh? If so, is there anyway we can move this and refactor over both files?

Reply:
I put this test in the `L0_launch_modes` and removed `L0_multi_gpu`.